doc/nrf/libraries/caf/ble_adv.rst

```diff
@@ -98,22 +63,29 @@ This lets you speed up finding the Peripheral by Bluetooth Centrals.
 
 Switching to slower advertising is done to reduce the energy consumption.
 
-Using Swift Pair
-================
-
-You can use the :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR` option to enable the Swift Pair feature.
-The feature simplifies pairing the Bluetooth Peripheral with Windows 10 hosts.
-
-.. note::
-   Make sure to add the Swift Pair data to advertising packets for unbonded device in the configuration file if you enable :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR` option.
-   The Swift Pair data must be added as the last member of ``ad_unbonded`` array.
-
 Power-down
 ==========
 
 When the system goes to the Power-down state, the advertising stops.
 
-If the Swift Pair feature is enabled with :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR`, the device advertises without the Swift Pair data for additional :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR_GRACE_PERIOD` seconds to ensure that the user does not try to connect to the device that is no longer available.
+Grace period
+------------
+
+If an advertising data provider requests non-zero grace period, the advertising stop on power-down is delayed by the requested time.
```

Comment:
```suggestion
If an advertising data provider requests non-zero grace period with :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR_GRACE_PERIOD`, the stopping of advertising on power-down is delayed by the requested time.
```

Reply:
The Kconfig option does not belong to an advertising data provider.
Changed `advertising stop` to `stopping of advertising`